nc-news-fe/api/getAllArticles.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios'
+
+const getAllArticles = () => {

```

Comment:
I don't understand why this is a file on it's own? ```export const getAllArticles = ()=>{}``` would allow more than one api call per file.